### PR TITLE
Login and logout

### PR DESCRIPTION
**Pull request by nguyenduyson11**
Ticket
 #34415 https://edu-redmine.sun-asterisk.vn/issues/34415
What PR do?
 Users can log into the account by email and log out of the account
Ticket bug in backlog
 none
Envident
image

### CODE REVIEW

**Comment by nguyenduyson11** [comment]
ready

**Comment by nguyenduyson11** [comment]
ready

File: app/views/layouts/_header.html.erb [hunk 1]
```diff
@@ -8,7 +8,11 @@
       <%= link_to "Tour", "/", class: "nav_item_link" %>
       <%= link_to "About", "/about", class: "nav_item_link" %>
       <%= link_to "Contact", "/contact", class: "nav_item_link" %>
-      <%= link_to "Login", "/login", class: "nav_item_link" %>
+      <% if logged_in? %>
+        <%= link_to "Logout", logout_path, method: :delete, class: "nav_item_link" %>
```
Comment (tham-1781) [inline app/views/layouts/_header.html.erb]: chưa i18n nè

File: config/routes.rb [hunk 2]
```diff
@@ -1,7 +1,11 @@
 Rails.application.routes.draw do
+  get 'sessions/new'
```
Comment (tham-1781) [inline config/routes.rb]: nháy đôi

**Comment by nguyenduyson11** [comment]
ready

File: app/views/layouts/_header.html.erb [hunk 3]
```diff
@@ -4,11 +4,15 @@
       <a href="" class="logo_nav_link">Travel</a>
     </div>
     <div class="nav_item">
-      <%= link_to "Home", "/", class: "nav_item_link" %>
-      <%= link_to "Tour", "/", class: "nav_item_link" %>
-      <%= link_to "About", "/about", class: "nav_item_link" %>
-      <%= link_to "Contact", "/contact", class: "nav_item_link" %>
-      <%= link_to "Login", "/login", class: "nav_item_link" %>
+      <%= link_to t("Home"), "/", class: "nav_item_link" %>
```
Comment (tham-1781) [inline app/views/layouts/_header.html.erb]: key của mấy cái này trong file .yml là "Home" à? (tat)

**Comment by nguyenduyson11** [comment]
ready

**Review by tham-1781 — COMMENTED** [review]
(merged)

**Pull request merged**